gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,5 +1,5 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
+distributionUrl=https\://services.gradle.org/distributions/gradle-4.6-bin.zip
```

Comment:
When have we lost the distribution checksum verification? ;)

Reply:
That seems to be a regression in https://github.com/mockito/mockito/commit/70a7cf8c17bcbf0ffb5316d6dd2457d9708bbe5f#diff-fd9d61e37d939c941e7a62077f38d62d Fixing :+1: